sdk/anomalydetector/ai-anomaly-detector-rest/src/anomalyDetectorRest.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getClient, ClientOptions } from "@azure-rest/core-client";
+import { KeyCredential } from "@azure/core-auth";
+import { AnomalyDetectorRestClient } from "./clientDefinitions";
+
+/**
+ * Initialize a new instance of the class AnomalyDetectorRestClient class.
+ * @param Endpoint type: string Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).
+ * @param ApiVersion type: string Anomaly Detector API version (for example, v1.1).
+ * @param credentials type: KeyCredential
+ */
+export default function createClient(
+  Endpoint: string,
+  ApiVersion: string,
+  credentials: KeyCredential,
+  options: ClientOptions = {}
+): AnomalyDetectorRestClient {
+  const baseUrl = options.baseUrl ?? `${Endpoint}/anomalydetector/${ApiVersion}`;
```

Comment:
We don't want to have ApiVersion as a positional parameter of `createClient` this should be:

```suggestion
export default function createClient(
  Endpoint: string,
  credentials: KeyCredential,
  options: ClientOptions & { apiVersion: string }= {}
): AnomalyDetectorRestClient {
  const apiVersion = options.apiVersion ?? "v1"
  const baseUrl = options.baseUrl ?? `${Endpoint}/anomalydetector/${apiVersion}`;
```

Please take this change, which is the same as in the PR I sent earlier. We have an open PR in codegen to fix this in there.